hack/unwanted-dependencies.json

```diff
@@ -144,6 +144,7 @@
       ],
       "github.com/mailru/easyjson": [
         "github.com/go-openapi/swag",
+        "github.com/google/pprof",
```

Comment:
weird that this showed up here... none of the vendored deps used this...

oof... this came in via https://github.com/google/pprof/commit/7a8ec2ada47b0ebe103b66ebb94efd558c8d9821#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R6 which added a headless chrome testing module and pulls in a bunch of random stuff :-(

Reply:
the rest of the PR looks fine... mind opening an issue in that repo asking them to isolate / drop the test-only github.com/chromedp/chromedp dep since it's propagating transitive dependencies to users of the library?

Reply:
I already did: https://github.com/google/pprof/issues/862

Reply:
updating to HEAD of pprof will drop out that dependency now since https://github.com/google/pprof/pull/866 merged 🎉